linux/sound.cpp

```diff
@@ -210,9 +207,11 @@ int CSound::Init ( const int /* iNewPrefMonoBufferSize */ )
     // without a Jack server, Jamulus makes no sense to run, throw an error message
     if ( bJackWasShutDown )
     {
-        throw CGenErr ( tr ( "The Jack server was shut down. This software "
-                             "requires a Jack server to run. Try to restart the software to "
-                             "solve the issue." ) );
+        throw CGenErr ( QString ( tr ( "The JACK server was shut down. %1 "
+                                       "requires a JACK server to run. Please restart %1 to "
+                                       "start JACK again. If the error persists, please review "
```

Comment:
Given the fact that "review" is extremely vague, I'd just remove it. 

```suggestion
                                       "start JACK again. "
```